### ISSUE TITLE
[HL1S] [Steampipe] Map c3a2d Bug 201

### ISSUE THREAD

**Issue by Pinhead-alejandro** [Theuaredead/Half-Life-Source-Update#205]
Well, this is probably related with the same problem than that one that made non solid bodies... however I think this HEV suits should be solid:

![2015-12-25_00085](https://cloud.githubusercontent.com/assets/13278870/12012174/80515e84-ace9-11e5-882e-a093180797ec.jpg)

![2015-12-25_00086](https://cloud.githubusercontent.com/assets/13278870/12012175/805202c6-ace9-11e5-8105-0e28504dbdcc.jpg)

![2015-12-25_00087](https://cloud.githubusercontent.com/assets/13278870/12012176/80543c62-ace9-11e5-85ff-fbea9c0eb69c.jpg)

**Comment by Theuaredead** [comment]
It appears that this was added into Half-Life: Source, as I do not see this HEV in Half-Life 1.
Pretty sure in Half-Life you could clip through a HEV when you already have one on.

**Comment by Pinhead-alejandro** [comment]
Yeah I understand all that, of course VALVe used just the HEV suit from the game, and yeah, since you have already a HEV suit you can't take it, (however, if you haven't one, you can use it, but it dosen't work like that one from C1A0D, since at c3a2d, you can only see appear the screen numbers but nothing more), but in this case is very ugly to can pass throw them, I could really prefer crash with an invisible wall or something, if try to made them solid is impossible or something like that. And yeah I know that you can't find HEV suits at this point in Half-Life. Is just one of all those differences that, like I said you, I am not reporting. But, in this case, I am just saying of make more perfect the player experience by avoiding this ugly situation. I mean, when I reach this place, I just can imagine that I will crash against a solid object.

However I was thinking... and maybe we should remove these suits... I mean, when I saw this in Half-Life, it was evident for me that other people traveled to Xen by using the suits from all these spaces... and for that reason we couldn't find any suit there. Now, someone thought it could be nice put this suits there, but I think it lose a little the complex concepts that we had when we saw this empty places. But, at the same time, I understand that since all the place is full of ammo, indicating they were looking for weapons and ammo to prepare the next travel to Xen, is a good point put these suits. But, I think it could be much better if we put MARK III suits, (the older suits with helmets), instead of the new suits MARK IV, (without helmets). Because I mean, is evident that lots of people traveled to Xen before Freeman with the same goal, destroy to Nihilanth. Having in mind, MARK IV is the new version, and not only that, it is very recent..., (please I know that there are a lot of people at forums arguing why Freeman's suit hasn't helmet, and speaking about the texture name and staff like that, however in Half-Life you can hear this sentence: "Ah! one of the new HEV suits, it will be very useful to you" so sorry, but I am totally sure that all those suits we found in Xen, with helmets, were Mark III, even more, in "Opposing Force" we can find a girl, without helmet, indicating that she had a MARK IV suit while the other guys was using MARK III with helmet, even more In Half-Life the girls hasn't helmets and Rosemberg congrats to Gina for the new suits even when Gina already had a MARK V prototype in production, (we can find this info in the Decay Letter). Therefore, if we assume that this people was fighting to find ammo, and Freeman was the last one in travel to Xen, I think is more logic find MARK III, instead of MARK IV, since is logic to imagine that the first travelers took the most newer and best suits, leaving the worse ones until the end, I could say more, I could remove one or two of these suits indicating that other people went to Xen before Freeman, and replace two or one of them with MARK III, indicating that Freeman found all the things discarded by early travelers plus the new weapons found in the last hours. I really think this thing willl help a lot to the immersion in the game. If you didn't understand something please ask and I will try to explain it in other way.
